.idea/runConfigurations/Debug_Elasticsearch.xml

```diff
@@ -1,11 +1,11 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Debug Elasticsearch" type="Remote">
     <option name="USE_SOCKET_TRANSPORT" value="true" />
-    <option name="SERVER_MODE" value="true" />
+    <option name="SERVER_MODE" value="false" />
```

Comment:
@danhermann Is this change intended? Also the one for `AUTO_RESTART`?

Reply:
@ywangd, no I have a PR open to revert it here: https://github.com/elastic/elasticsearch/pull/58992

Reply:
Thanks